src/gate/index.js

```diff
@@ -54,21 +57,46 @@ Object.assign(UserGate.prototype, {
     return this._matchesSample(user) || this._matchesList(user);
   },
 
+  allowsUniform: function(user) {
```

Comment:
Actually if we make a major release anyway per my comments about Bower then we could just remove `sampleCharacterSet` (still wouldn't be breaking per se since it'd just be ignored—what I mean is that I wouldn't even feel the need to keep it in docs)

Reply:
Purely to avoid a breaking change - I thought it'd be nice to support older usage of this library even after these changes (given our internal usage for send gating) - although per https://github.com/mixmaxhq/user-gate/pull/5#discussion_r281297105 and other comments in this PR, we can't avoid making a breaking change.

Reply:
>  replacing _matchesSample with _matchesUniformSample

https://github.com/mixmaxhq/user-gate/pull/5/commits/7e6dfa937363205c20409112b4cd4f9e5553baf0